notebooks/05_operationalize/create_and_configure_cluster.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
Meta question: what are you hoping users will learn / take away from this notebook? Why did you chose to make the Databricks creation a notebook instead of wrapping the functionality in reco_utils and calling it via the O16N notebook?

Reply:
The O16N notebook is intended to be executed on databricks. This notebook is intended to create the resource that can do that. The user should gain the ability to with running 1 notebook, create a configured cluster to run the 016N notebook on. Newer versions of databricks (the new 5.1 version specifically) allows for installing libraries from inside notebooks, but to leverage that, we need to know whether the cosmosdb connector jar works on it.

https://docs.databricks.com/release-notes/runtime/5.1.html

Reply:
Once we get the runtime version issue figured out, then I think there's a reasonable question of which one is easier?

1. Logging into the ADB workspace and manually creating an appropriate cluster, installing all dependencies - the three pypi libraries, the java jar, and the reco_utils.egg (which you need to manually create), and uploading the o16n notebook.
2. Logging into the ADB workspace to generate a personal access token, and then running this (or a similar) notebook to do do all of the steps in 1.

Reply:
favoring the CLI approach in the other Pull request, and focusing this one on just the notebook and setup instructions in SETUP.md for operationalization.